### DISCUSSION TITLE
when we update version 2.0.16, we cannot see any buttons and it's not working

### DISCUSSION THREAD

**Discussion by fatabek** [kimai/kimai#4014] [General] [answered]
### Describe the issue

When we upgraded to version 2.0.16, none of the buttons worked. No icons appear.

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

2.0.16

### How do you run Kimai?

Virtual Server or alike

### Which PHP version are you using?

8.1

### Logfile

No log files

### Screenshots

![Kimai](https://user-images.githubusercontent.com/41823969/236588247-99ff656d-cc8f-40f7-a492-d08a14266815.JPG)

**Comment by kevinpapst** [comment]
you are missing assets. let me guess: you are using docker?

**Reply by fatabek** [reply]
No, not using docker (sorry for my english)

**Reply by kevinpapst** [reply]
then check your browser console (right click "inspect element") and check what the error is.
i assume something is caching or the directory `public/build/` wasn't properly update

**Comment by RajendraWdipl** [comment]
i am also having the same issue in version 2.0.16. Icons are not visible just like the above shown screenshot.  I am using shared hosting which has a Public_html folder into which i have uploaded this project. 

browser console shows some error related to fonts. Kindly guide me
![image](https://user-images.githubusercontent.com/128778333/236805111-2de975df-99f3-4267-9c72-9544d0a8f807.png)

**Reply by kevinpapst** [reply]
Kimai cannot run in a subdirectory, that is not supported. 
You need to use it via something like https://kimai.example.com/ and not https://www.example.com/kimai/

**Reply by RajendraWdipl** [reply]
there is no subdirectory.  our website address is just like https://kimai.example.com/ and when i hit it goes to https://kimai.example.com/public/en/login

**Reply by kevinpapst** [reply, marked as answer]
It's all in the documentation. Re-read the install chapter. 
You need to configure `public/` as webroot, it might not appear in the URL!

**Comment by fatabek** [comment]
It's works now, thank's a lot. But I would like to make a small note that until version 1.13.11 this was working without editing.

**Reply by kevinpapst** [reply]
Even though the documentation always said the opposite: that it is not supported. So you were lucky 😁 

Exposing `public/` in the URL also means someone can call yourdomain/.env.ini and (if you didn't add a rule to prevent that) see your database credentials in plaintext. Which would be a terrible security issue.
